src/getDefaultConfig.ts

```diff
@@ -0,0 +1,41 @@
+import { http, cookieStorage, createConfig, createStorage } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { coinbaseWallet } from 'wagmi/connectors';
+import type { AppConfig } from './types';
+
+// getDefaultConfig returns a custom WagmiConfig (https://wagmi.sh/react/api/createConfig) using OnchainKit provided settings.
+// This function is used if the user does not provide WagmiProvider as a parent in the React context.
+export const getDefaultConfig = ({
+  apiKey,
+  config,
+}: {
+  apiKey?: string;
+  config?: AppConfig;
+}) => {
+  return createConfig({
+    chains: [base, baseSepolia],
+    connectors: [
+      coinbaseWallet({
+        appName: config?.appearance?.name || 'My OnchainKit App',
+        appLogoUrl:
+          config?.appearance?.logo ||
+          'https://onchainkit.xyz/favicon/48x48.png?v4-19-24',
+        preference: 'smartWalletOnly',
+      }),
+    ],
+    storage: createStorage({
+      storage: cookieStorage,
+    }),
+    ssr: true,
```

Comment:
my understanding is this is broken for ssr frameworks like `Next.js` with the default value 

```
Turning on the ssr property means that content from the external stores will be hydrated on the client after the initial mount.
```

for non-SSR apps, is there any drawback to this approach? cc: @alessey @alissacrane-cb 


[link](https://wagmi.sh/react/guides/ssr)